test/unit/shipment_packer_test.rb

```diff
@@ -36,6 +36,19 @@ def test_divide_order_with_single_line_into_two_packages
     end
   end
 
+  def test_divide_order_with_single_line_into_two_packages_max_weight_as_float
+    max_weight = 68038.8555
+
+    items = [{:grams => 45359, :quantity => 2, :price => 1.0}]
```

Comment:
I'd advocate for colon syntax! I don't really mind if it's inconsistent within a test file, because when we do come around to amend the rest of the file, it's fewer changes. I could be convinced otherwise though. 😛